### ISSUE TITLE
Default config incorrect?

### ISSUE THREAD

**Issue by philipzaengle** [zaengle/craft-conventions#27]
The example config includes the following resolve config
```
        'resolver' => [
            'class' => DefaultResolver::class,
            'settings' => [],
        ],
```

Which causes an error to be thrown. Removing the resolver seems to solve the error.

**Comment by ostark** [comment]
Namespace?

**Comment by tomdavies** [comment]
fixed in Dare 39aecd499ae651daabddf5ad94a2c73d9a8aba2b
